<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SemSeg</class>
 <widget class="QMainWindow" name="SemSeg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SemSeg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_semseg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>semseg</string>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>20</y>
      <width>761</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openobj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Open .obj</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
